mqtt/mqtt-broker/src/broker.rs

```diff
@@ -1981,6 +2019,222 @@ pub(crate) mod tests {
         assert_matches!(sub_rx.try_recv(), Err(TryRecvError::Empty))
     }
 
+    #[tokio::test]
+    async fn test_notify_single_connection() {
```

Comment:
some tests checking sessions to publication data gathering move to `sys_topic` module